planemo/training/tutorial.py

```diff
@@ -347,6 +347,11 @@ def init_from_existing_tutorial(self, tuto_name):
         # get the data library
         self.init_data_lib()
 
+        # create the FAQ page
+        if not os.path.exists(os.path.join(self.faq_dir, 'index.md')):
```

Comment:
```suggestion
        os.makedirs(self.faq_dir)
        if not os.path.exists(os.path.join(self.faq_dir, 'index.md')):
```

Also it seems to me this should be part of the `create_tutorial()` instead of `init_from_existing_tutorial()` ?

Reply:
good call